packages/admin/README.md

```diff
@@ -61,12 +61,12 @@ Please, refer to the implementation for more details.
 
 ## Developer commands
 
-### `derive-sponsor-wallet-address`
+### `derive-sponsor-wallet`
 
-Derives the address of the wallet designated by an Airnode for a requester. You need to specify the xpub but if it is not provided then this command will try to fetch it from the AirnodeRrp contract.
+Derives a sponsorWallet designated by an Airnode for a sponsor and returns the address of the wallet. You need to specify the xpub but if it is not provided then this command will try to fetch it from the AirnodeRrp contract.
```

Comment:
I am not a fan of `sponsorWallet` in the README description. Why not just `sponsor wallet`?